.github/workflows/integration.yml

```diff
@@ -258,15 +258,16 @@ jobs:
             test/nri/nri.test
       - run: scripts/github-actions-packages
       - run: scripts/github-actions-setup
-      - name: Switch to crun instead of runc
-        if: ${{ matrix.run.defaultRuntime == 'crun' }}
+      - name: Switch to runc instead of crun
+        if: ${{ matrix.run.defaultRuntime == 'runc' }}
         run: |
-          sudo ln -sf $(command -v crun) $(command -v runc)
-          runc --version
+          sudo ln -sf $(command -v runc) $(command -v crun)
+          crun --version
       - name: Run the test suite
         run: sudo -E test/test_runner.sh
         env:
           JOBS: ${{ matrix.run.jobs }}
           RUN_CRITEST: ${{ matrix.run.critest }}
           RUNTIME_TYPE: ${{ matrix.run.runtimeType }}
           TEST_USERNS: ${{ matrix.run.userns }}
+          CONTAINER_DEFAULT_RUNTIME: ${{ matrix.run.defaultRuntime }}
```

Comment:
Do we need this?

Edit: Ah looking at the integration test changes below we probably do. :thinking:

Reply:
I think it's more idiomatic. The copying of binaries and spoofing is a little weird IMO (though I still attempted to do so just in case)